src/stripe-payment.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * `stripe-payment`
+ * `Payment broker for stripe`
+ *
+ * @demo demo/index.html
+ * @customElement `stripe-payment`
+ */
+import { LitElement, html, css } from "lit-element/lit-element.js";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+class StripePayment extends LitElement {
+  /**
+   * Convention
+   */
+  static get tag() {
+    return "stripe-payment";
+  }
+  /**
+   * LitElement render styles
+   */
+  static get styles() {
+    return [
+      css`
+        [hidden] {
+          display: none !important;
+        }
+        :host {
+          font-family: "Nunito Sans", -apple-system, ".SFNSText-Regular",
+            "San Francisco", BlinkMacSystemFont, "Segoe UI", "Helvetica Neue",
+            Helvetica, Arial, sans-serif;
+          align-items: center;
+          display: grid;
+          grid-gap: 12px;
+          grid-template-areas:
+            "support support"
+            "stripe submit"
+            "output output";
+        }
+        stripe-payment-request {
+          grid-area: submit/stripe/stripe/submit;
+        }
+        stripe-elements {
+          grid-area: stripe;
+        }
+        mwc-button {
+          grid-area: submit;
+        }
+        json-viewer {
+          grid-area: output;
+        }
+
+        stripe-elements:not(:defined),
+        json-viewer:not(:defined),
+        mwc-button:not(:defined) {
+          display: none;
+        }
+      `
+    ];
+  }
+  /**
+   * LitElement / popular convention
+   */
+  static get properties() {
```

Comment:
let's use babel decorators for these

and maybe let's extend from StripeBase? hmm not quite sure about that one TBH

Reply:
Added this too